extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -177,6 +181,9 @@ export class AmpIframe extends AMP.BaseElement {
    * @override
    */
   onLayoutMeasure() {
+    this.isLikeFixedAd_ = isAdLike(this.element) &&
```

Comment:
This looks good, but needs to be renamed now since it combines more than just "ad-like" behavior. It could be, for instance, `isDisallowedAsAd_` or some such.